### PR TITLE
Add GithubClient to config

### PR DESCRIPTION
**Pull request by meichend**

### CODE REVIEW

File: src/config.ts [hunk 1]
```diff
@@ -15,13 +17,17 @@ export function getInputConfiguration(): configuration {
 		return _.merge({}, moduleDefaults[module.type], module);
 	});
 	console.log("Using module config: ", moduleOptions);
+	const accessToken = getInput("accessToken", { required: true });
```
Comment (derrickpersson) [inline src/config.ts]: I think you can just get `GITHUB_TOKEN` from the env, I don't know if you need to name this a specific variable.

File: action.yml [hunk 2]
```diff
@@ -23,6 +23,9 @@ inputs:
     description: 'A prefix for the commit message'
     required: false
     default: '*chore*'
+  GITHUB_TOKEN:
+    description: 'Github token with write permissions to the repository'
+    required: false
```
Comment (derrickpersson) [inline action.yml:28]: Still needed?

**Review by derrickpersson — APPROVED** [review]
One question, other than that LGTM

File: action.yml (hunk 2 above)
Comment (davidscodex) [reply action.yml:28]: Yes. The action can't access the secrets without it passed in as a variable.

**Review by davidscodex — APPROVED** [review]

**Comment by davidscodex** [comment]
nit: it's probably best to delete `src/git.ts` it's been replaced by the new client.

**Pull request merged**